apps/condo/domains/organization/gql.js

```diff
@@ -1,14 +1,14 @@
 const { gql } = require('graphql-tag')
-const { genTestGQLUtils } = require('@core/keystone/gen.gql.utils')
+const { generateGQLUtils } = require('@core/keystone/gen.gql.utils')
```

Comment:
This file should also be generated by createschema. You can try to delete it and generate by --force